js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
+      this.firstWord = firstWord;
+    }
+    else {
+      this.firstWord = 'Fizz';
+    }
+
+    if (secondWord !== '') {
+      this.secondWord = secondWord;
+    }
+    else {
+      this.secondWord = 'Buzz';
+    }
+
+    console.log(this.firstWord);
+
+  }
+
+  fizzBuzz.prototype.readArray = function(lowNumber, highNumber) {
+
+    for (var i = 0; i < (highNumber - lowNumber + 1); i++) {
```

Comment:
I like how you're variably setting the upper-limit of your `for` loop, but keep in mind that this calculation will be performed with every iteration of your loop. A more efficient solution would be to perform that calculation and store it in a variable:

```javascript
var ceiling = highNumber - lowNumber + 1;
for (var i = 0; i < ceiling; i++) { /* ... */ }
```